### PR TITLE
Scaffold & style archive editorial

### PR DESCRIPTION
**Pull request by varffs**
also adds sitewide layout logic (introduces style flash which will need resolution)

### CODE REVIEW

**Comment by mikeyrayvon** [comment]
style flash?

**Comment by varffs** [comment]
Well as the js can't execute before the page starts to render things move around. I think we could possibly add a fade in of everything other than header but i dunno

**Comment by varffs** [comment]
We can drop that js approach but it will require fixed heights for these vertically centered layouts so we will have to target specific screen sizes and it wont really be responsive

File: archive-editorial.php [hunk 1]
```diff
@@ -4,16 +4,16 @@
 
 <!-- main content -->
 
-<main id="main-content">
+<main id="main-content" class="container">
```
Comment (mikeyrayvon) [inline archive-editorial.php:7]: in some templates `.container` is on `<main>` and others on `<section>` whats the logic?

Comment (varffs) [inline archive-editorial.php:7]: no logic was retroactively fixing other templates and didn't realise the container was in the wrong place

File: css/site.styl [hunk 2]
```diff
@@ -431,7 +431,8 @@ img
   height: 100%
 
 #archive-editorial-titles
-  max-width: 500px
+// >>> be nice if this value had a relation to container or something, or to other content like this. It could also be an indent not width centered
+  max-width: 350px
```
Comment (mikeyrayvon) [inline css/site.styl]: i think this should have `margin-left: $margin-mid` instead of max-width and centered

Comment (varffs) [inline css/site.styl]: thats fine by me. I wasnt sure how to interpret the designs.

**Comment by mikeyrayvon** [comment]
i think we need a `max-height`on this puppy

screen shot 2016-08-15 at 2 34 04 pm

and probably a 

```
#footer
  background-color: $color-white
```

for good measure

**Comment by varffs** [comment]
You mean max height on the image? Yes it does. 100% will do it

**Comment by varffs** [comment]
But the footer should not be overlapping like that. Perhaps better to put overflow: hidden on the main-container-holder

**Comment by mikeyrayvon** [comment]
are you doing mobile first on all this?

**Comment by varffs** [comment]
no. is that the plan? i was going to add styles but there was not base styles to work from

**Comment by mikeyrayvon** [comment]
well site.styl is mobile, then there are responsive styls for tablet and desktop. ive been first doing layout for the mobile design, and then following with tablet and desktop

**Comment by mikeyrayvon** [comment]
sorry if i didnt make that clear :/

**Comment by mikeyrayvon** [comment]
can you fix the height issue @varffs ?

**Comment by varffs** [comment]
I can but not til later/tomorrow

**Comment by mikeyrayvon** [comment]
doesnt seem to be working 
screen shot 2016-08-17 at 10 02 32 am

File: css/site.styl [hunk 3]
```diff
@@ -429,12 +432,19 @@ img
...
+  .col
+    height: 100%
+    min-height: 60px
 
 #archive-editorial-titles
   margin-left: $margin-basic
 
-.archive-editorial-image:not(:first-of-type)
-  display: none
+.archive-editorial-image
```
Comment (mikeyrayvon) [inline css/site.styl]: these should be hidden on mobile. see mobile pdf pg6

Comment (varffs) [inline css/site.styl]: it is

**Comment by varffs** [comment]
I do actually have that issue with still breaking layout tho which is pretty annoying. I'm trying to make a generic solution that doesnt set hard heights on anything more than one element but maybe i have to go dirtier :(

**Comment by mikeyrayvon** [comment]
i think you gotta go dirtier 💪

**Comment by mikeyrayvon** [comment]
this was the wrong branch but its fine

**Comment by mikeyrayvon** [comment]
your doing `if (_this.$mainContent.height() < _this.windowHeight) {` but why would the mainContent be less than the window hight if were not setting the height on the images?  why dont we just set the max-height of the images instead??

File: css/responsive/tablet.styl [hunk 4]
```diff
@@ -41,11 +41,8 @@ for col in 1..$cols
 // ARCHIVE EDITORIAL
 
 #posts-archive-editorial
-  height: 100%
-  min-height: 600px
   .col
-    height: 100%
-    min-height: 600px
+    min-height: 620px
```
Comment (mikeyrayvon) [inline css/responsive/tablet.styl:45]: whats up with this?  how did u come to 620?

Comment (varffs) [inline css/responsive/tablet.styl:45]: that is the max height of the image size used for the hover images

Comment (mikeyrayvon) [inline css/responsive/tablet.styl:45]: ok just to understand how youre approaching this: the logic is such that if the browser height doesnt contain `headerHeight + footerHeight + 620` then...what happens?

Comment (mikeyrayvon) [inline css/responsive/tablet.styl:45]: maybe its a question for clay?

Comment (varffs) [inline css/responsive/tablet.styl:45]: there isnt anymore funky stuff in this feature branch. this is just forcing 620 height so smaller images still vertically center (I hope)

Comment (varffs) [inline css/responsive/tablet.styl:45]: its not important. if you want to just remove all these styles thats fine

Comment (mikeyrayvon) [inline css/responsive/tablet.styl:45]: cause i would think the minheight would be less than that. at least to fit the footer on my screen without having it fixed

Comment (varffs) [inline css/responsive/tablet.styl:45]: well the images can be as big as 620px high so if they are and they are not scaled [no scaling or funk happening in this branch] then the page will have to overflow window

**Pull request merged**